models/leaderboard/getEventLeaderboard.js

```diff
@@ -0,0 +1,21 @@
+const { pool } = require('../db');
+/**
+ * @param {String} param0.eventname
+ * @return {Promise}
+ */
+function getEventLeaderboard(eventname) {
+  return new Promise((resolve, reject) => {
+    pool.query(
+      `SELECT name,email,phone,rank FROM users_events_map ORDER BY rank ASC LIMIT 5 WHERE event=?`,
```

Comment:
There will be no column called rank in users_events_map. Go to events table..there is scores column. Which contains all top 3 or 5 scores separated by comma. Respond with that value